lib/tasks/mastodon.rake

```diff
@@ -33,6 +33,58 @@ namespace :mastodon do
     end
   end
 
+  module AvatarHeader
```

Comment:
- not idiomatic 
-> I want isolate model and rake.
- not work with S3
-> I don't use cloud storage
-> fix to your simple way